scripts/lib/CIME/SystemTests/pet.py

```diff
@@ -30,6 +30,10 @@ def _case_one_setup(self):
             if self._case.get_value("NTHRDS_%s"%comp) <= 1:
                 self._case.set_value("NTHRDS_%s"%comp, 2)
 
+        # Subtle: machines like titan will not let you overload the number of threads
```

Comment:
I don't understand why there was any overloading going on - the PET uses full nodes on the first test and undersubscribes on the second one.   Isn't this just a matter of updating the aprun command correctly for each of (between) the two runs?